plugin-scalebar/build.gradle.kts

```diff
@@ -13,7 +13,9 @@ android {
     targetSdkVersion(AndroidVersions.targetSdkVersion)
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
   }
-
+  kotlinOptions {
+    jvmTarget = "1.8"
```

Comment:
why is it needed?

Reply:
Needed here: https://github.com/mapbox/mapbox-maps-android/pull/856/files#diff-f5b05909ec597796a068b2e9299e1a2c78f80fc016d6b50e016fa950394d183eR59